Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -496,6 +496,7 @@
 
 ### LS2 - MWGR ###
 steps['RunCosmics2020']={'INPUT':InputInfo(dataSet='/ExpressCosmics/Commissioning2019-Express-v1/FEVT',label='2020GR0',run=[334393],events=100000,location='STD')}
```

Comment:
Hello @boudoul, shall we remove this step if you wanna replace with RunCosmics2020GEM completely? Otherwise it isn't used elsewhere.

Reply:
Hi @chayanit , sorry I overlooked your comment. As mentioned in the description , I would like to keep as it is the express WF (138.2 using RunCosmics2020 ) in order to keep a run to be tested in IBs with the Strip Tracker. The newly introduced run in the prompt WF (138.1 using RunCosmics2020GEM ) doesn't contain the strip tracker (but GEM)

Reply:
OK fair enough @boudoul